.github/workflows/updatecli.yaml

```diff
@@ -2,10 +2,13 @@
 name: updatecli
 
 on:
+  # Allow to be run manually
   workflow_dispatch:
   schedule:
-    # Run every 15minutes
-    - cron: '*/15 * * * *'
+    # Run once per week (to avoid alert fatigue)
+    - cron: '0 2 * * 1' # Every monday at 2am UTC
```

Comment:
Why 2am specifically?

Reply:
- Early in the EU morning so if it breaks anything we can fix it early in our day
- Not really in the weekend to avoid unsollicited notifications